reports/tests/graphs/tests_sleep_pattern.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+from zoneinfo import ZoneInfo
```

Comment:
We still support Python 3.7 and 3.8 and I'd like to keep it that way for as long as those versions are supported ([for another 2+ years](https://devguide.python.org/#status-of-python-branches)). Rather than use backports, I wonder if there is some other way to configure this test?

Reply:
Yup, done - switched to using `timezone` from datetime.